src/uiMainApp.py

```diff
@@ -142,9 +148,9 @@ def __init__(self, parent):
         # Hide the dev3201Window and dev2101Window
         self.vboxdl.Hide(self.dev3201Pan)
         self.vboxdl.Hide(self.dev2101Pan)
+        self.vboxdl.Hide(self.dev3141Pan)
```

Comment:
Line 151 - By default we show dev3141 panel in mainwindow, here we are showing 2301 as default panel. Change the line as self.vboxdl.Hide(self.dev2301Pan)